src/mcp/index.ts

```
@@ -0,0 +1,322 @@
+          method: req.method,
+        });
+        res.status(500).json({
+          error: "Internal Server Error",
+          message: err.message,
+        });
+      },
+    );
+
+    // Add async error handling wrapper
+    const asyncHandler =
+      (fn: Function) =>
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)